packages/neuron-wallet/src/startup/sync-block-task/sync.ts

```diff
@@ -44,19 +44,18 @@ export const switchNetwork = async (url: string, genesisBlockHash: string, chain
   // listen to address created
   addressCreatedSubject.subscribe(async (addresses: Address[]) => {
     if (blockListener) {
-      const infos: LockHashInfo[] = (await Promise.all(
-        addresses.map(async addr => {
-          const hashes: string[] = await LockUtils.addressToAllLockHashes(addr.address)
-          // undefined means false
-          const isImporting: boolean = addr.isImporting === true
-          return hashes.map(h => {
-            return {
-              lockHash: h,
-              isImporting,
-            }
-          })
+      const lockUtils = new LockUtils(await LockUtils.loadSystemScript(url))
```

Comment:
And here.

Reply:
After changes from ed71600 the only places having `loadSystemScript` is `sync-block-task/indexer`. Should that use `systemScript(url)` instead as well?

Reply:
If in offline mode, it will switch to normal sync, so when switch to indexer, the network would be available, so I think replace or not is not important.

Reply:
I see. Thanks.